src/components/App/PuzzleGenerate/index.tsx

```diff
@@ -0,0 +1,101 @@
+import * as React from 'react';
+import { useEffect, useState } from 'react';
+import styles from './PuzzleGenerate.module.scss';
+import { Button, Input } from '../../Global';
+import { createPuzzleInstance, getAllPuzzles } from '../../../services';
+
+const PuzzleGenerate = () => {
+  const [puzzleList, setPuzzleList] = useState([]);
+  const [hint, setHint] = useState('');
+  const [latitude, setLatitude] = useState('');
+  const [longitude, setLongitude] = useState('');
+  const [address, setAddress] = useState('');
+  const [puzzleId, setPuzzleId] = useState('');
+
+  const handleSubmit = () => {
+    console.log(hint, latitude, longitude, address, puzzleId);
```

Comment:
remove log

Reply:
This log is a placeholder for another feature!